app/paths.js

```diff
@@ -22,9 +22,13 @@ module.exports = {
   user: {
     logIn: '/login',
     profile: '/my-profile',
+    twoFactorAuth: {
+      index: '/my-profile/two-factor-auth',
+      configure: '/my-profile/two-factor-auth/configure',
+      complete: '/my-profile/two-factor-auth/complete'
```

Comment:
where is this used?

Reply:
Ha good spot, an earlier version had a complete page but settled on redirect to profile